newm-server/src/main/kotlin/io/newm/server/features/user/database/UserTable.kt

```diff
@@ -33,4 +33,5 @@ object UserTable : UUIDTable(name = "users") {
     val companyName: Column<String?> = text("company_name").nullable()
     val companyLogoUrl: Column<String?> = text("company_logo_url").nullable()
     val companyIpRights: Column<Boolean?> = bool("company_ip_rights").nullable()
+    val admin: Column<Boolean> = bool("admin")
```

Comment:
Maybe it's safer here to do: `bool("admin").default(false)`

Reply:
Fixed this and cleaned up code it was being manually set up elsewhere.